src/transformers/models/patchtst/modeling_patchtst.py

```diff
@@ -1962,10 +1965,10 @@ def forward(
             past_observed_mask=past_observed_mask,
             output_hidden_states=output_hidden_states,
             output_attentions=output_attentions,
-            return_dict=return_dict,
+            return_dict=True,
         )
         # get output head. y_hat is of shape [bs x num_targets] or tuple of this shape
-        y_hat = self.head(model_output[0])
+        y_hat = self.head(model_output.last_hidden_state)
```

Comment:
mmm this seems like a regression to me ?

Reply:
so my question was if the call to the model is internal, can we set the dict to something convenient and then output the dict as per the forward or shall we pass the return dict to the internal model?

Reply:
@kashif I think it is much clearer to use `last_hidden_state` while enforcing `return_dict` in the internal call. Externally, the `forward` method still honors the `return_dict` passed in by the caller.

Reply:
Fine with me, but then the class should not even support return dict no?

Reply:
Ok